assets/src/components/propertiesPanel/status_run_block_tabs.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs"
+
+interface Props {
+  statusContent: JSX.Element
+}
+
+const TabStatusIcon = () => (
+  <svg className="m-tabs__tab-status-icon" width="14" height="14">
+    <circle
+      className="m-tabs__tab-status-icon-outer-circle"
+      cx="7"
+      cy="7"
+      r="6"
+    />
+    <circle
+      className="m-tabs__tab-status-icon-inner-circle"
+      cx="7"
+      cy="7"
+      r="4"
+    />
+  </svg>
+)
+
+const StatusRunBlockTabs = ({ statusContent }: Props) => (
+  <div className="m-tabs">
+    <Tabs className="m-tabs__tabs">
```

Comment:
The wrapping div is redundant. This could be a single `<Tabs className="m-tabs">`.